src/Sepes.Infrastructure/Util/AzureResourceProivisoningTimeoutResolver.cs

```diff
@@ -8,23 +8,23 @@ public static int GetTimeoutForOperationInSeconds(string resourceType, string op
         {
             if(resourceType == AzureResourceType.StorageAccount)
             {
-                return 100;
+                return 180;
             }
             else if (resourceType == AzureResourceType.NetworkSecurityGroup)
             {
-                return 100;
+                return 180;
             }
             else if (resourceType == AzureResourceType.VirtualNetwork)
             {
-                return 100;
+                return 180;
             }
             else if (resourceType == AzureResourceType.ResourceGroup)
             {
                 return 30;
             }
             else if (resourceType == AzureResourceType.Bastion)
             {
-                return 300;
+                return 600;
```

Comment:
Does these values just come from testing?

Reply:
Yes, just observations on how long these operations tend to take. I saw that for Bastion, 300(5 mins) was sometimes to short, so de queue item became visible again. This caused it to be processed by another thread while the old thread was still on it